strictdoc/backend/sdoc/models/document_view.py

```diff
@@ -0,0 +1,29 @@
+from typing import List, Optional
+
+from strictdoc.backend.sdoc.models.type_system import (
+    ViewElementTags,
+    ViewElementHiddenTag,
+)
+
+from strictdoc.helpers.auto_described import auto_described
+
+
+@auto_described()
+class ViewElement:
+    def __init__(
+        self,
+        id: str,
+        tags: List[ViewElementTags],
+        hidden_tags: Optional[List[ViewElementHiddenTag]] = None,
+        name: Optional[str] = None,
+    ):
+        self.id = id
```

Comment:
Please include the type annotations for the fields too:

`self.view_id: str = view_id`

This extra verbosity pays off when the static analyzers can give you helpful hints if you happen to use these types incorrectly somewhere else in the code. The developer guide mentions this.

I will not comment on the similar issues below. Please double-check this for all `self.*` fields in the PR.